src/colmap/controllers/incremental_pipeline.cc

```diff
@@ -270,6 +284,93 @@ bool IncrementalPipeline::LoadDatabase() {
   return true;
 }
 
+void IncrementalPipeline::Reconstruct(
```

Comment:
Unfortunately it's hard to carefully diff this function because it was moved - consider reverting the move to make it easier to review and catch unintended changes.